frontend/packages/pipelines-plugin/integration-tests/testData/installPipelineOperator1.8.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: CatalogSource
+metadata:
+  name: custom-operators
+  namespace: openshift-marketplace
+spec:
+  sourceType: grpc
+  image: quay.io/openshift-pipeline/redhat-operator-index:236551
```

Comment:
This might change now and then, visit the url to get the latest image at the time of testing. (you could add as a comment)

Reply:
Added the comment